docs/reference/fields.txt

```diff
@@ -99,7 +99,6 @@ You can safely omit type specifications when:
 
 Types that are not supported as dynamic attributes since they cannot be cast are:
 
-- ``BigDecimal``
```

Comment:
Is there a test where a BigDecimal value is written to & read from a dynamic field? I'm expecting no based on this not being possible previously.

Reply:
Could be done in a separate PR.

Reply:
Not that I can find, adding one in the MONGOID-4237 PR

Reply:
done